### ISSUE TITLE
Capturing pending payments fails against PaymentIntents

### ISSUE THREAD

**Issue by christophersjchow** [stripe/stripe-mirakl-connector#46]
With the latest version 1.2.6, the call to capture pending payments fails as the charge was created by a PaymentIntent. Here are the logs I see.

```
23:37:01 INFO      [messenger] App\Message\CapturePendingPaymentMessage was handled successfully (acknowledging to transport). ["message" => App\Message\CapturePendingPaymentMessage { …},"class" => "App\Message\CapturePendingPaymentMessage"]
[2020-12-14 23:37:01] messenger.INFO: Received message App\Message\CapturePendingPaymentMessage {"message":"[object] (App\\Message\\CapturePendingPaymentMessage: {})","class":"App\\Message\\CapturePendingPaymentMessage"} []
23:37:01 INFO      [messenger] Received message App\Message\CapturePendingPaymentMessage ["message" => App\Message\CapturePendingPaymentMessage { …},"class" => "App\Message\CapturePendingPaymentMessage"]
[2020-12-14 23:37:01] app.INFO: [Stripe API] Call to Charge/Capture {"stripeId":"ch_1HwdFjGTPbVNSl93erQs7zrd","amount":400} []
23:37:01 INFO      [app] [Stripe API] Call to Charge/Capture ["stripeId" => "ch_1HwdFjGTPbVNSl93erQs7zrd","amount" => 400]
[2020-12-14 23:37:01] app.ERROR: Could not capture Stripe Payment: This uncaptured Charge was created by a PaymentIntent (pi_1HwdF0GTPbVNSl93Lk4HqKnU). You must capture the PaymentIntent instead. For more information, see https://stripe.com/docs/payments/payment-intents/creating-payment-intents#separate-auth-capture. {"paymentId":"ch_1HwdFjGTPbVNSl93erQs7zrd","amount":400,"stripeErrorCode":null} []
23:37:01 ERROR     [app] Could not capture Stripe Payment: This uncaptured Charge was created by a PaymentIntent (pi_1HwdF0GTPbVNSl93Lk4HqKnU). You must capture the PaymentIntent instead. For more information, see https://stripe.com/docs/payments/payment-intents/creating-payment-intents#separate-auth-capture. ["paymentId" => "ch_1HwdFjGTPbVNSl93erQs7zrd","amount" => 400,"stripeErrorCode" => null]
[2020-12-14 23:37:01] messenger.INFO: Message App\Message\CapturePendingPaymentMessage handled by App\Handler\CapturePendingPaymentHandler::__invoke {"message":"[object] (App\\Message\\CapturePendingPaymentMessage: {})","class":"App\\Message\\CapturePendingPaymentMessage","handler":"App\\Handler\\CapturePendingPaymentHandler::__invoke"} []
23:37:01 INFO      [messenger] Message App\Message\CapturePendingPaymentMessage handled by App\Handler\CapturePendingPaymentHandler::__invoke ["message" => App\Message\CapturePendingPaymentMessage { …},"class" => "App\Message\CapturePendingPaymentMessage","handler" => "App\Handler\CapturePendingPaymentHandler::__invoke"]
[2020-12-14 23:37:01] messenger.INFO: App\Message\CapturePendingPaymentMessage was handled successfully (acknowledging to transport). {"message":"[object] (App\\Message\\CapturePendingPaymentMessage: {})","class":"App\\Message\\CapturePendingPaymentMessage"} []
23:37:01 INFO      [messenger] App\Message\CapturePendingPaymentMessage was handled successfully (acknowledging to transport). ["message" => App\Message\CapturePendingPaymentMessage { …},"class" => "App\Message\CapturePendingPaymentMessage"]
```

The changes in #38 means only the charge's ID is stored in the database. How would it be possible now to capture authorisations done as per the guide here: https://stripe.com/docs/payments/capture-later

**Comment by youssef-stripe** [comment]
Thanks for flagging @christophersjchow. We will fix this and make sure the capture flow is compatible with PaymentIntents.

**Comment by youssef-stripe** [comment]
This is fixed in v1.2.7.
